src/PyNvCodec/src/PyNvEncoder.cpp

```diff
@@ -455,6 +477,10 @@ void Init_PyNvEncoder(py::module& m)
       .def("Format", &PyNvEncoder::GetPixelFormat,
            R"pbdoc(
         Return encoded video stream pixel format.
+    )pbdoc")
```

Comment:
Can you update the stubs or remind us to do it after merging? https://github.com/NVIDIA/VideoProcessingFramework/blob/ec8caf75c341b9f4a125db59676c6742c8b666fd/Makefile#LL41

Reply:
Sure, let me generate stubs and push them

Reply:
we should really automate this. e.g. via https://github.com/stefanzweifel/git-auto-commit-action

Reply:
taking a look